site/app/views/forum/ForumThreadView.php

```diff
@@ -766,7 +789,7 @@ public function createPost($thread_id, $post, $unviewed_posts, $function_date, $
         $thread_dir = FileUtils::joinPaths(FileUtils::joinPaths($this->core->getConfig()->getCoursePath(), "forum_attachments"), $thread_id);
 
         $date = DateUtils::parseDateTime($post["timestamp"], $this->core->getConfig()->getTimezone());
-        if (!is_null($post["edit_timestamp"])) {
+        if (isset($post["edit_timestamp"]) && !is_null($post["edit_timestamp"])) {
```

Comment:
Can `!empty()` be used here?

Reply:
No, as per what I read from https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/

`isset( )` is probably the correct one here

Reply:
This whole conditional can be replaced with `if(!empty())` no need to have isset and !is_null. Empty doesn't trigger an error if the variable doesn't exist.